.travis.yml

```diff
@@ -45,8 +45,38 @@ matrix:
       install: echo "MAVEN_OPTS='-Xmx3000m'" > ~/.mavenrc && mvn install -q -ff -DskipTests -B
       before_script:
         - unset _JAVA_OPTIONS
+
       script: echo "MAVEN_OPTS='-Xmx512m'" > ~/.mavenrc && mvn test -B -Pparallel-test -Dmaven.fork.count=2 -pl '!processing,!server'
 
+      # processing module test with SQL Compatible Null Handling
+    - sudo: false
+      env:
+        - NAME="processing module test"
```

Comment:
Give different names in this group (currently names repeat the prev. group)

Reply:
don.